ui-admin/src/study/notifications/NotificationContent.tsx

```diff
@@ -1,30 +1,68 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
+import React, { useEffect, useState } from 'react'
+import { NavLink, Outlet, Route, Routes, useNavigate } from 'react-router-dom'
 import NotificationConfigTypeDisplay, { deliveryTypeDisplayMap } from './NotifcationConfigTypeDisplay'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faPlus } from '@fortawesome/free-solid-svg-icons/faPlus'
 import { StudyEnvContextT } from '../StudyEnvironmentRouter'
+import NotificationConfigView from './NotificationConfigView'
+
+const CONFIG_GROUPS = [
+  { title: 'Event', type: 'EVENT' },
+  { title: 'Reminder', type: 'TASK_REMINDER' },
+  { title: 'Ad-hoc', type: 'AD_HOC' }
+]
 
 /** shows configuration of notifications for a study */
 export default function NotificationContent({ studyEnvContext }: {studyEnvContext: StudyEnvContextT}) {
   const currentEnv = studyEnvContext.currentEnv
-  return <div className="container m-3 p-3 bg-white">
-    <h2 className="h4">Participant Notifications</h2>
-    <div className="flex-grow-1 p-3">
-      <ul className="list-unstyled">
-        { currentEnv.notificationConfigs.map(config => <li key={config.id} className="p-1">
-          <div className="d-flex">
-            <Link to={`configs/${config.id}`}>
-              <NotificationConfigTypeDisplay config={config}/>
-              <span className="detail"> ({deliveryTypeDisplayMap[config.deliveryType]})</span>
-            </Link>
-          </div>
-        </li>
-        ) }
-      </ul>
-      <button className="btn btn-secondary" onClick={() => alert('not yet implemented')}>
-        <FontAwesomeIcon icon={faPlus}/> Add
-      </button>
+  const navigate = useNavigate()
+  const [prevEnv, setPrevEnv] = useState<string>(currentEnv.environmentName)
```

Comment:
I think this name is unclear. Not sure if this is "previous environment" or "preview environment".